types/lib/model.d.ts

```diff
@@ -170,14 +175,14 @@ export interface WhereOperators {
    *  - `[Op.like]: '%hat',` becomes `LIKE '%hat'`
    *  - `[Op.like]: { [Op.any]: ['cat', 'hat']}` becomes `LIKE ANY ARRAY['cat', 'hat']`
    */
-  [Op.like]?: string | Literal | AnyOperator | AllOperator;
+  [Op.like]?: string | Literal | AnyOperator | AllOperator | ValuesOperator;
```

Comment:
Is this actually correct?
The only reference in the code I can see is within an `Op.all` context.